modules/Bio/EnsEMBL/Variation/DBSQL/LDFeatureContainerAdaptor.pm

```diff
@@ -153,7 +152,6 @@ sub vcf_executable {
 sub temp_path {
```

Comment:
Suggest adding some documentation for sub temp_path including some of what in PR description.
When is $TMP_PATH set? 
Would other calls be expecting to be /tmp